src/bin/pgcopydb/main.c

```diff
@@ -278,6 +280,27 @@ set_logger()
 	/* set our logging facility to use our semaphore as a lock mechanism */
 	(void) log_set_udata(&log_semaphore);
 	(void) log_set_lock(&semaphore_log_lock_function);
+
+	/* duplicate output to log file if PGCOPYDB_LOG env var is set.
+	 * Warning: if the value is incorrect, it may cause segfault.
+	*/
+	if (env_exists("PGCOPYDB_LOG"))
+	{
+		char env_log_file[BUFSIZE] = { 0 };
+
+		if (get_env_copy("PGCOPYDB_LOG", env_log_file, BUFSIZE) > 0)
+		{
+			if ((log_fp = fopen(env_log_file, "w"))) /* extra parenthesis around assignment to avoid compiler warnings */
+			{
+				log_set_fp(log_fp);
+			}
+			else
+			{
+				log_error("Failed to open log file for writing %s. Error reason: %s", env_log_file, strerror(errno));
+				exit(EXIT_CODE_BAD_ARGS);
+			}
+		}
+	}
```

Comment:
Please remove that parts, the corresponding feature has been added in the meantime already. See https://pgcopydb.readthedocs.io/en/latest/ref/pgcopydb_clone.html#environment and PGCOPYDB_LOG_TIME_FORMAT, PGCOPYDB_LOG_JSON, PGCOPYDB_LOG_FILENAME, and PGCOPYDB_LOG_JSON_FILE.